### ISSUE TITLE
Early Stopping

### ISSUE THREAD

**Issue by bala1802** [gmihaila/ml_things#7]
Can you please add examples for earlystopping based on the loss metrics during the training?

**Comment by gmihaila** [comment]
Hi @bala1802 thank you for your question!

Here is a link form HuggingFace that let's you use early stopping: https://huggingface.co/transformers/main_classes/callback.html#transformers.EarlyStoppingCallback

Also, you could use PyTorchLightning: https://pytorch-lightning.readthedocs.io/en/latest/common/early_stopping.html
